tests/functional/fx_fennec_v1_sign_up.js

```diff
@@ -58,6 +58,20 @@ define([
         })
 
         .then(FunctionalHelpers.testIsBrowserNotified(self, 'fxaccounts:can_link_account'))
```

Comment:
@vladikoff - I added a new FunctionalHelpers function `noSuchBrowserNotification` that ensures a notification has *not* happened yet. I've added it here (and in a few other places).